cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -126,23 +135,59 @@ class CallSubqueryTest extends DocumentingTest {
         )))) { resultTable() }
     }
 
-    section("Aggregation and side-effects", "subquery-aggregation") {
-      p("""Subqueries can be useful to do aggregations for each row and to isolate side-effects.
-          #This example query creates five `Clone` nodes for each existing person.
-          #The aggregation ensures that cardinality is not changed by the subquery.
-          #Without this, the result would be five times as many rows.""".stripMargin('#'))
+    section("Cardinality and aggregation", "subquery-aggregation") {
+      p(
+        """Returning subqueries change the cardinality of the query: The results of each subquery call are collected and together they give the result of the
+          |enclosing query.""".stripMargin)
+      p("""The following example lists each connected person that each person has:""")
       query("""MATCH (p:Person)
               #CALL {
-              #  UNWIND range(1, 5) AS i
-              #  CREATE (c:Clone)
-              #  RETURN count(c) AS numberOfClones
+              #  WITH p
+              #  MATCH (p)-[:FRIEND_OF]-(c:Person)
+              #  RETURN c.name AS friend
               #}
-              #RETURN p.name, numberOfClones""".stripMargin('#'),
+              #RETURN p.name, friend""".stripMargin('#'),
       ResultAssertions(r => r.toSet should equal(Set(
-          Map("p.name" -> "Alice", "numberOfClones" -> 5),
-          Map("p.name" -> "Bob", "numberOfClones" -> 5),
-          Map("p.name" -> "Charlie", "numberOfClones" -> 5),
-          Map("p.name" -> "Dora", "numberOfClones" -> 5)
+          Map("p.name" -> "Alice", "friend" -> "Bob"),
+          Map("p.name" -> "Bob", "friend" -> "Alice"),
+        )))) { resultTable() }
+      p(
+        """The number of results of the subquery changed the number of results of the enclosing query: Instead of 4 rows, one for each node), there are
+          |now 2 rows which were found for Alice and Bob respectively. The nodes for Charlie and Dora are not returned.""".stripMargin)
```

Comment:
Let's be explicit, even though it is a sad truth :cry: 
```suggestion
          |now 2 rows which were found for Alice and Bob respectively. No rows are returned for Charlie and Dora since they have no friends.""".stripMargin)
```